cadc-access-control/src/main/java/ca/nrc/cadc/ac/client/GMSClient.java

```diff
@@ -89,6 +89,8 @@
 import javax.net.ssl.SSLSocketFactory;
 import javax.security.auth.Subject;
 
+import ca.nrc.cadc.auth.*;
+import ca.nrc.cadc.net.*;
```

Comment:
I'm guessing your IDE changes the imports to * when there are more than 4 references :)  I prefer to see the full list mostly for consistency in this project.